python/equistore-operations/equistore/operations/lstsq.py

```diff
@@ -148,7 +148,7 @@ def _lstsq_block(X: TensorBlock, Y: TensorBlock, rcond, driver) -> TensorBlock:
     Y_n_properties = Y.values.shape[-1]
     Y_values = Y.values.reshape(-1, Y_n_properties)
 
-    _check_same_gradients(X, Y, props=None, fname="lstsq")
+    _check_same_gradients_raise(X, Y, props=None, fname="lstsq")
```

Comment:
The doc tests don't like it when I set `props=["samples", "components", "properties"]` but pass with `props=None`. @PicoCentauri please may you just verify for me that this is definitely what we want in the context of least squares?

Reply:
The reason was that checking for `"properties"` does not make sense for `lstsq`. The values of X are a matrix while for Y it is only a vector.

Reply:
> I also think we could set props=["samples", "components", "properties"] as the default to the function instead of None. What do you think?

if we have a default it should be a tuple, not a list (i.e. `props=("samples", "components", "properties")`), because mutable default values are problematic in Python. Otherwise, this sounds like a good idea to me!

Reply:
Yes, yes indeed. I will change this.